src/content/docs/cheerpj3/11-guides/CheerpJ-explanation.mdx

```diff
@@ -0,0 +1,81 @@
+---
+title: CheerpJ components
+description: How does CheerpJ works?
```

Comment:
This should be "how does CheerpJ work?" but I think we can do better since it's obvious this page is about CheerpJ.

I'd suggest (in order of my preference):

- Architecture
- How it works
- Technical details